examples/components/CCFRAUD/traininsilo/run.py

```diff
@@ -598,7 +598,7 @@ def run(args):
     )
     trainer.execute(args.checkpoint)
 
-    if torch.cuda.is_available() or int(os.environ["WORLD_SIZE"]) > 1:
```

Comment:
let's handle a case if there's no `WORLD_SIZE` env var and apply the changes across all examples.

Reply:
yep could we use os.environ.get("WORLD_SIZE", 1) so it would work even if the env var is not set?

Reply:
(I actually don't remember if the variable is set when the world size is actually just 1 and the job is not distributed)